api/src/main/java/jakarta/data/repository/Update.java

```diff
@@ -24,51 +24,21 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-
 /**
- * <p>Annotates a repository method to perform update operations.</p>
+ * <p>Lifecycle annotation for repository methods which perform update operations.</p>
  *
- * <p>The {@code Update} annotation indicates that the annotated repository method requests that one or more entities
- * be updated if found in the database. To request updates to specific entity instances, the annotated
- * repository method must have a single parameter whose type must be one of the following:
+ * <p>The {@code Update} annotation indicates that the annotated repository method updates the state of one or more
+ * entities already held in the database. The annotated repository method must have a exactly one parameter whose type
```

Comment:
This will preserve the future flexibility to use the Update annotation for query-based updates as well as allow vendors to experiment with those approaches in the mean time,
```suggestion
 * entities already held in the database. To supply entity instances to the method,
 * the annotated repository method must have a exactly one parameter whose type
```